src/batch-client/index.ts

```diff
@@ -628,9 +628,22 @@ export class ZimbraBatchClient {
 
 			if (folders.linkedFolders) {
 				folders.linkedFolders = folders.linkedFolders.map((folder: any) => {
-					if (folder.view === FolderView.Message) {
-						const { absFolderPath, oname, folders } = folder;
-
+					if (
+						folder.view === FolderView.Message ||
+						folder.view === FolderView.Contact
+					) {
+						const {
+							absFolderPath,
+							oname,
+							folders,
+							ownerZimbraId,
+							sharedItemId
+						} = folder;
+
+						if (folder.view === FolderView.Contact) {
```

Comment:
Let's add comment on why we're doing this for 1st level of linkedFolders only.

Reply:
Added comment